gossip/client/hubclient_test.go

```diff
@@ -597,6 +597,11 @@ func TestGetSourceKeys(t *testing.T) {
 			body:     fmt.Sprintf(`{"entries":[{"id":"https://a-log.com","pub_key":"%s","kind":"TRILLIANSLR"}]}`, b64(pubKeyDER)),
 			wantKind: "TRILLIANSLR",
 		},
+		{
+			desc:     "Valid Trillian SMR Source",
+			body:     fmt.Sprintf(`{"entries":[{"id":"https://a-log.com","pub_key":"%s","kind":"TRILLIANSMR"}]}`, b64(pubKeyDER)),
```

Comment:
a-map.com ?

Reply:
The tests all share the same source URL, so changed it to `source.example.com`